healthcheck/omnect_health__services.sh

```diff
@@ -96,9 +96,7 @@ function do_get_infos() {
 	fi
 	do_rate "$rating"
 	print_info_header "${ME}(${service})" "$rating"
-	if [    "$rating" != 0
-	     -a -r /var/run/omnect_health_log/${service}.exit-info
-	   ]; then
+	if [ "$rating" != 0 -a -r /var/run/omnect_health_log/${service}.exit-info ]; then
```

Comment:
how do we want to handle that? is this an increase of the omnect-device-service patch version? imho, yes. it's not rust code, but part of the component.

Reply:
you are right, this should result in a new patch version.
I adapted cargo files accordingly.